src/explore-education-statistics-common/src/components/ContentHtml.module.scss

```diff
@@ -0,0 +1,25 @@
+@import '~govuk-frontend/govuk/base';
+
+.infoIcon {
+  background-color: $govuk-link-colour;
+  border-radius: 10px;
+  color: govuk-colour('white') !important;
```

Comment:
Any particular reason we're using `!important` quite often in here? If it's to do with the usage of links, we'll probably find that we can change this more easily by using a button element for this instead.

Reply:
They were necessary to apply the styles when it was a link - but now I've changed it to a button that aren't necessary anymore, so I've removed them.